ocs_ci/ocs/constants.py

```diff
@@ -2862,3 +2865,10 @@
 
 # ascii characters
 ENTER_KEY = chr(13)
+
+# helper script to perform meta data operations [create, rename, change permissions...etc] on app pod to fill MDS cache
+METAIO = os.path.join(TEMPLATE_WORKLOAD_DIR, "helper_scripts/meta_data_io.py")
+# helper script to perform file creation IO on app pod to fill MDS cpu
+FILE_CREATOR_IO = os.path.join(
```

Comment:
Not a blocker for this PR, but IMO we should push this script to an image and run it as a container